mlflow/pytorch/__init__.py

```diff
@@ -395,7 +406,16 @@ def _load_model(path, **kwargs):
     else:
         model_path = path
 
-    return torch.load(model_path, **kwargs)
+    if Version(pytorch_version) >= Version('1.5.0'):
+        return torch.load(model_path, **kwargs)
+    else:
+        try:
+            # load the model as an eager model.
+            return torch.load(model_path, **kwargs)
+        except Exception: # pylint: disable=W0703
```

Comment:
Can you replace `W0703` to `catching-too-general-exception`?

Reply:
sure